src/nunit.analyzers.playground/nunit.analyzers.playground.csproj

```diff
@@ -1,68 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\packages\NUnit.3.10.1\build\NUnit.props" Condition="Exists('..\packages\NUnit.3.10.1\build\NUnit.props')" />
```

Comment:
I had to remove this line and the `Target` element below (with the name `EnsureNuGetPackageBuildImports`) otherwise I got complaints about "This project references NuGet package(s) that are missing on this computer. ...The missing file is ..\packages\NUnit.3.10.1\build\NUnit.props." when compiling.